verifier/verifier.go

```diff
@@ -526,3 +547,15 @@ func verifyX509TrustedIdentities(certs []*x509.Certificate, trustPolicy *trustpo
 
 	return fmt.Errorf("signing certificate from the digital signature does not match the X.509 trusted identities %q defined in the trust policy %q", trustedX509Identities, trustPolicy.Name)
 }
+
+func logVerificationResult(logger log.Logger, result *notation.ValidationResult) {
+	if result.Error == nil {
+		return
+	}
+	switch result.Action {
+	case trustpolicy.ActionLog:
+		logger.Warnf("%v validation failed with validation action set to \"logged\". Failure reason: %v", result.Type, result.Error)
+	case trustpolicy.ActionEnforce:
+		logger.Errorf("%v validation failed. Failure reason: %v", result.Type, result.Error)
```

Comment:
FYI, result.Type starts with lower case letters. If we want to capitalize all the logs, we should do it here as well.

Reply:
Updated.